apiary.apib

```diff
@@ -991,9 +991,247 @@ Outputs the list of peers that were removed.
 
 # Group log
 
-## level
+Interact with the daemon log output
 
-## tail
+'ipfs log' contains utility commands to affect or read the logging
+output of a running daemon.
+
+This command cannot be called directly.
+
+## level [POST /log/level{?arg1,arg2}]
+'ipfs log level' is a utility command used to change the logging
+output of a running daemon.
+
++ Parameters
+    + arg1 (string) - The subsystem logging identifier. Use `all` for all subsystems.
+        + Members
+            - bitstwap
+            - blockstore
+            - dht
+            - merkledag
+            - all
+    + arg2 (enum[string])
+        + Members
+            - debug
+            - info
+            - warning
+            - error
+            - fatal
+            - panic
+
++ Request Without Arguments
+
+    #### curl
+
+        curl -i -X POST "http://localhost:5001/api/v0/log/level"
+
+    + Body
+
+        ```
+        curl -i -X POST "http://localhost:5001/api/v0/log/level"
+        ```
+
++ Response 400
+
+    + Headers
+
+        ```
+        Date: Fri, 05 Feb 2016 21:12:32 GMT
+        Content-Length: 32
+        Content-Type: text/plain; charset=utf-8
+        ```
+
+    + Attributes (string)
+
+    + Body
+
+        ```
+        Argument 'subsystem' is required
+        ```
+
++ Request With Empty Argument
+
+    The response is the same if the first argument is invalid. For example:
+
+        curl -i -X POST "http://localhost:5001/api/v0/log/level?arg=kitten"
+
+    #### curl
+
+        curl -i -X POST "http://localhost:5001/api/v0/log/level?arg="
+
+    + Body
+
+        ```
+        curl -i -X POST "http://localhost:5001/api/v0/log/level?arg="
+        ```
+
++ Response 400
+
+    + Headers
+
+        ```
+        Date: Fri, 05 Feb 2016 21:13:04 GMT
+        Content-Length: 28
+        Content-Type: text/plain; charset=utf-8
+        ```
+
+    + Attributes (string)
+
+    + Body
+
+        ```
+        Argument 'level' is required
+        ```
+
++ Request With Invalid Subsystem Argument
+
+    #### curl
+
+        curl -i -X POST "http://localhost:5001/api/v0/log/level?arg=kitten&arg=info"
+
+    + Body
+
+        ```
+        curl -i -X POST "http://localhost:5001/api/v0/log/level?arg=kitten&arg=info"
+        ```
+
++ Response 500
+
+    + Headers
+
+        ```
+        Content-Type: application/json
+        Trailer: X-Stream-Error
+        Transfer-Encoding: chunked
+        Date: Fri, 05 Feb 2016 21:22:27 GMT
+        Transfer-Encoding: chunked
+        ```
+
+    + Attributes (Error)
+        - Message: "Error: No such logger"
+        - Code: 0
+
+    + Body
+
+        ```
+        {
+          "Message": "Error: No such logger",
+          "Code": 0
+        }
+        ```
```

Comment:
I think apiary will be fine if you leave out the body if all attributes have examples